tests/foreman/cli/test_capsule.py

```diff
@@ -20,7 +20,7 @@ def setUp(self):
 
     @run_only_on('sat')
     @tier1
-    def test_verify_redmine_3875(self):
+    def test_positive_create_with_random_url(self):
```

Comment:
I'd remove `random` as we have many tests using random values but we don't explicitly call it out.